### PR TITLE
Feature/70133

### PR DESCRIPTION
**Pull request by pawel-snowdog**

### CODE REVIEW

File: Magento_GiftRegistry/templates/address/edit.phtml [hunk 1]
```diff
@@ -1,11 +1,14 @@
 <?php
-
-// phpcs:disable Magento2.Templates.ThisInTemplate
-// phpcs:disable Magento2.Files.LineLength.MaxExceeded
-// phpcs:disable Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace
-// phpcs:disable Squiz.WhiteSpace.ScopeClosingBrace.ContentBefore
-
+use Magento\Customer\Block\Widget\Name;
+?>
+<?php
```
Comment (anqaka) [inline Magento_GiftRegistry/templates/address/edit.phtml:4]: maybe we can keep it in one `<?php ?>` tag?

File: Magento_Catalog/templates/product/list.phtml [hunk 2]
```diff
@@ -37,21 +49,11 @@ use Magento\Framework\App\Action\Action;
      */
     $pos = $block->getPositioned();
     ?>
-    <a
-        href="#layered-filter-block"
-        class="skip-nav"
-        data-mage-init='{"skipNav": {} }'
-    >
-        <?= $block->escapeHtml(__('Go to filters')); ?>
-    </a>
```
[the remark this answers is not in the document]
Comment (pawel-snowdog) [reply Magento_Catalog/templates/product/list.phtml:52]: something no/yes happend here ;)

File: Magento_Catalog/templates/product/list.phtml [hunk 3]
```diff
@@ -37,21 +49,11 @@ use Magento\Framework\App\Action\Action;
...
 
-    <?php /** @var $_product \Magento\Catalog\Model\Product */ ?>
 
+    <?php /** @var $_product \Magento\Catalog\Model\Product */ ?>
     <?php if ($viewMode === 'products-list'): ?>
-        <ul
-            class="list"
-            id="products-list"
-        >
```
[the remark this answers is not in the document]
Comment (pawel-snowdog) [reply Magento_Catalog/templates/product/list.phtml:54]: yep - I don't know how it was removed

File: Magento_Catalog/templates/product/list.phtml [hunk 4]
```diff
@@ -304,10 +308,7 @@ use Magento\Framework\App\Action\Action;
             <?php endforeach; ?>
         </ul>
     <?php else: ?>
-        <ul
-            class="catalog-grid"
-            id="products-list"
-        >
```
[the remark this answers is not in the document]
Comment (pawel-snowdog) [reply Magento_Catalog/templates/product/list.phtml:310]: yep - I don't know how it was removed

File: Magento_Catalog/templates/product/list/items.phtml [hunk 5]
```diff
@@ -183,16 +193,19 @@ $sliderResponsiveConfig = '[
...
+                data-shuffle-weighted="<?= /* @noEscape */ $isWeightedRandom ?>"
             >
         <?php else: ?>
             <section
-                class="section <?= $class ?>"
+                class="section <?= $block->escapeHtmlAttr($class) ?>"
                 data-mage-init='{ "upsellProducts": {} }'
-                data-limit="<?= $limit ?>"
+                data-limit="<?= $block->escapeHtmlAttr($limit) ?>"
                 data-shuffle="<?= $shuffle ?>"
```
[the remark this answers is not in the document]
Comment (pawel-snowdog) [reply Magento_Catalog/templates/product/list/items.phtml:206]: fixed

File: Magento_Catalog/templates/product/list/items.phtml [hunk 6]
```diff
@@ -202,6 +215,7 @@ $sliderResponsiveConfig = '[
         <div class="section__heading">
             <h2
                 id="block-<?= $class ?>-heading"
+                role="heading"
```
[the remark this answers is not in the document]
Comment (pawel-snowdog) [reply Magento_Catalog/templates/product/list/items.phtml:218]: not needed

File: Magento_Catalog/templates/product/list/items.phtml [hunk 7]
```diff
@@ -329,49 +348,32 @@ $sliderResponsiveConfig = '[
...
-                                                            name="product"
-                                                            value="<?= /* @noEscape */ (int)$_item->getEntityId() ?>"
-                                                        >
-                                                        <input
-                                                            type="hidden"
-                                                            name="<?= /* @noEscape */ Magento\Framework\App\Action\Action::PARAM_NAME_URL_ENCODED ?>"
-                                                            value="<?= /* @noEscape */ $postParams ?>"
-                                                        >
-                                                        <?= $block->getBlockHtml('formkey') ?>
+                                                    <?php if (!$_item->getTypeInstance()->isPossibleBuyFromList($_item)):?>
```
[the remark this answers is not in the document]
Comment (pawel-snowdog) [reply Magento_Catalog/templates/product/list/items.phtml:363]: fixed

File: Magento_CatalogWidget/templates/product/widget/content/grid.phtml [hunk 8]
```diff
@@ -201,9 +199,7 @@ use Magento\Framework\App\Action\Action;
                                             <?php endif; ?>
 
                                             <?php if ($block->getAddToCompareUrl() && $showCompare): ?>
-                                                <?php //phpcs:disable ?>
                                                 <?php $compareHelper = $this->helper(\Magento\Catalog\Helper\Product\Compare::class);?>
```
[the remark this answers is not in the document]
Comment (pawel-snowdog) [reply Magento_CatalogWidget/templates/product/widget/content/grid.phtml:202]: fixed

File: Magento_Catalog/templates/product/list/items.phtml [hunk 9]
```diff
@@ -428,4 +480,15 @@ $sliderResponsiveConfig = '[
             <?= $sliderBlockAfter->toHtml(); ?>
         </div>
     </section>
+    <?php if (!$block->isRedirectToCartEnabled() && $_item):?>
```
[the remark this answers is not in the document]
Comment (pawel-snowdog) [reply Magento_Catalog/templates/product/list/items.phtml:483]: fixed

File: Magento_Customer/templates/form/edit.phtml [hunk 10]
```diff
@@ -253,41 +262,52 @@
...
+            }
+        },
+        ignore: ':hidden:not(' + ignore + ')'
+    script;
+else:
+    $scriptString .= <<<script
+        ignore: ignore ? ':hidden:not(' + ignore + ')' : ':hidden'
+    script;
+endif;
+$scriptString .= <<<script
```
[the remark this answers is not in the document]
Comment (pawel-snowdog) [reply Magento_Customer/templates/form/edit.phtml:297]: imo it's ok bc it's after "endif" not in "if" statement

**Pull request merged**